### ISSUE TITLE
wrong command suggestion

### ISSUE THREAD

**Issue by lefred** [datacharmer/dbdeployer#140]
**Describe the bug**
When trying a wrong command, a suggested command is prompted but it's not valid.

**To Reproduce**
1. Run the command `dbdeployer downloads get-by-version 5.7.21` 
2. Read the message 
3. Run the command provided from the message above: `dbdeployer downloads get-remote mysql 5.7.21 linux`

**Expected behavior**
The returned suggestion should work ;)

If applicable, copy and paste the command and the output. Do not use screen shots, unless they add value to the report.  Text that can be edited is preferred to graphics.
```
[fred@letoshiba ~] $ dbdeployer downloads get-by-version 5.7.21
Error getting version 5.7.21 (mysql-linux)[minimal: false - newest: false - guess: false]: error detecting latest version
Try using 'dbdeployer downloads get-remote mysql 5.7.21 linux'

[fred@letoshiba ~] $ dbdeployer downloads get-remote mysql 5.7.21 linux
Manages remote tarballs

Usage:
  dbdeployer downloads [command]

Available Commands:
  add            Adds a tarball to the list
  add-remote     Adds a tarball to the list, by searching MySQL downloads site 
  export         Exports the list of tarballs to a file
  get            Downloads a remote tarball
  get-by-version Downloads a remote tarball
  get-unpack     Downloads and unpacks a remote tarball
  import         Imports the list of tarballs from a file or URL
  list           list remote tarballs
  reset          Reset the custom list of tarballs and resume the defaults
  show           Downloads a remote tarball
  tree           Display a tree by version of remote tarballs

Flags:
  -h, --help   help for downloads

Global Flags:
      --config string           configuration file (default "/home/fred/.dbdeployer/config.json")
      --sandbox-binary string   Binary repository (default "/home/fred/opt/mysql")
      --sandbox-home string     Sandbox deployment directory (default "/home/fred/sandboxes")
      --shell-path string       Path to Bash, used for generated scripts (default "/usr/bin/bash")
      --skip-library-check      Skip check for needed libraries (may cause nasty errors)

Use "dbdeployer downloads [command] --help" for more information about a command.
```

**Environment:**
 - OS: Fedora 34
 - dbdeployer 1.63.0

**Comment by datacharmer** [comment]
Thanks for the report.
Yep. It should be `add-remote` instead of `get-remote`. Easy to fix

**Comment by datacharmer** [comment]
Fixed in [release 1.63.1](https://github.com/datacharmer/dbdeployer/releases/tag/v1.63.1)
